Competition2018/src/main/java/competition/subsystems/elevator/commands/MoveToMaxHeightCommand.java

```diff
@@ -13,26 +13,30 @@
 
     ElevatorSubsystem elevator;
     PIDManager pid;
-
-    double max = 90;
+    double max;
```

Comment:
this doesn't need to be a class level variable, since it is only used in `execute()`

Reply:
done